hoomd/mpcd/ReverseNonequilibriumShearFlowUtilities.h

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2009-2024 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+/*!
+ * \file ReverseNonequilibriumShearFlowUtilities.h
+ * \brief Helper functions for Reverse nonequilibrium shear flow
+ */
+
+#ifndef MPCD_REVERSE_NONEQUILIBRIUM_SHEAR_FLOW_UTILITIES_H_
+#define MPCD_REVERSE_NONEQUILIBRIUM_SHEAR_FLOW_UTILITIES_H_
+
+#ifdef NVCC
+#define HOSTDEVICE __host__ __device__ __forceinline__
+#else
+#define HOSTDEVICE
```

Comment:
We do like this in the geometry files, so I would do the same here:
```suggestion
#define HOSTDEVICE __host__ __device__ inline
#else
#define HOSTDEVICE inline __attribute__((always_inline))
```